src/lib/apputils/net-server.c

```diff
@@ -889,9 +893,15 @@ setup_addresses(struct socksetup *data)
                                  _("Failed setting up a %s socket (for %s)"),
                                  bind_type_names[addr.type],
                                  paddr(ai->ai_addr));
-                goto cleanup;
+                if (ret != EAFNOSUPPORT)
```

Comment:
On IPv4 systems, when `ret == EAFNOSUPPORT`, won't this result in "Failed setting up" messages being logged?  This seems potentially confusing since they may not be fatal.